fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1501,6 +1501,15 @@ def get_shard_from_optim_state_dict(self, full_optim_state_dict: Dict[str, Any])
         return full_optim_state_dict
 
 
+def _get_default_compute_device(module: nn.Module) -> torch.device:
```

Comment:
By my reading, this will never return `cpu`, so should be called `_get_default_cuda_device`